packages/neos-ui-ckeditor5-bindings/src/manifest.config.js

```diff
@@ -27,16 +26,21 @@ import RemoveFormat from '@ckeditor/ckeditor5-remove-format/src/removeformat';
 const addPlugin = (Plugin, isEnabled) => (ckEditorConfiguration, options) => {
     // we duplicate editorOptions here so it would be possible to write smth like `$get('formatting.sup')`
     if (!isEnabled || isEnabled(options.editorOptions, options)) {
-        ckEditorConfiguration.plugins = ckEditorConfiguration.plugins || [];
-        return $add('plugins', Plugin, ckEditorConfiguration);
+        return {
+            ...ckEditorConfiguration,
+            plugins: [
+                ...(ckEditorConfiguration.plugins ?? []),
+                Plugin
+            ]
+        };
     }
     return ckEditorConfiguration;
 };
```

Comment:
Highlighting this `addPlugin` utility function for everyone migrating `$add` to normal js